packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -310,17 +310,22 @@ class FlutterPlugin implements Plugin<Project> {
                 localEngineSrcPath this.localEngineSrcPath
                 targetPath target
                 previewDart2 previewDart2Value
-                strongMode strongModeValue	
+                strongMode strongModeValue
                 preferSharedLibrary preferSharedLibraryValue
                 sourceDir project.file(project.flutter.source)
                 intermediateDir project.file("${project.buildDir}/${AndroidProject.FD_INTERMEDIATES}/flutter/${variant.name}")
                 extraFrontEndOptions extraFrontEndOptionsValue
                 extraGenSnapshotOptions extraGenSnapshotOptionsValue
             }
 
+            Task deleteOutputDir = project.tasks.create(name: "deleteOutputDirectory${variant.name.capitalize()}", type:Delete) {
+                delete variant.mergeAssets.outputDir
+            }
```

Comment:
I believe there is a `cleanXxx` task for every output-producing task `xxx`, so this should not be necessary. See below.

Reply:
Cool, good to know.